### PR TITLE
Fixes polling latest runs in hydrator detailed view for metrics, logs and runs count

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Reduces the poll interval for polling latest runs from 10 seconds to 2 seconds.
- Modifies request to poll for logs and metrics based on latest run and program type.

### CODE REVIEW

File: cdap-ui/app/features/adapters/controllers/tabs/metrics-ctrl.js [hunk 1]
```diff
@@ -18,72 +18,99 @@ angular.module(PKG.name + '.feature.adapters')
...
+              var key = split[2] + '.' + split[3];
 
-          angular.forEach(metrics.series, function (metric) {
-            var split = metric.metricName.split('.');
-            var key = split[2] + '.' + split[3];
+              if (!metricObj[key]) {
+                metricObj[key] = {
+                  nodeType: split[1],
+                  nodeName: split[2],
+                  stage: +split[3]
```
Comment (anwar6953) [inline cdap-ui/app/features/adapters/controllers/tabs/metrics-ctrl.js]: whats the plus symbol for?

Comment (ajainarayanan) [inline cdap-ui/app/features/adapters/controllers/tabs/metrics-ctrl.js]: oops. Typo. Will remove it.

**Comment by astewes** [comment]
:+1: other than comment from @anwar6953  LGTM

**Comment by anwar6953** [comment]
LGTM

**Pull request merged**